package.json

```diff
@@ -1,26 +0,0 @@
-{
```

Comment:
Olá, tudo bem? Irei fazer a análise do seu código... Qualquer dúvida que tiver a respeito da mesma, mande uma mensagem no slack que irei explicar da melhor maneira possível!

Lembrando que este pull request (PR) não é para ser aceito, você pode deletá-lo ou simplesmente ignorá-lo...

Resultado da revisão...

Seu código não tem muitos problemas, eu só recomendaria vc mudar esses commits "last of the day" para algo que corresponda o que vc está commitando